commons-mongo/src/main/scala/com/avsystem/commons/mongo/scala/GenCodecCollection.scala

```diff
@@ -0,0 +1,13 @@
+package com.avsystem.commons
+package mongo.scala
```

Comment:
Please do not create packages named `scala`. This way we won't need all this `_root_` bullshit.